StripeConnect/StripeConnect/Source/Helpers/StripeConnectBundleLocator.swift

```diff
@@ -7,9 +7,15 @@
 
 import Foundation
 @_spi(STP) import StripeCore
- 
+
 final class StripeConnectBundleLocator: BundleLocatorProtocol {
     static let internalClass: AnyClass = StripeConnectBundleLocator.self
     static let bundleName = "StripeConnectBundle"
     static let resourcesBundle = StripeConnectBundleLocator.computeResourcesBundle()
 }
+
+extension BundleLocatorProtocol {
+    static var bundleVersion: String? {
+        resourcesBundle.infoDictionary?["CFBundleVersion"] as? String
+    }
+}
```

Comment:
See my comment in `ConnectWebView` – CFBundleVersion isn't reliable, so we shouldn't add this extension